jconstraints-core/src/main/java/gov/nasa/jpf/constraints/smtlibUtility/SMTProblem.java

```diff
@@ -22,19 +22,20 @@
 import gov.nasa.jpf.constraints.api.Expression;
 import gov.nasa.jpf.constraints.api.SolverContext;
 import gov.nasa.jpf.constraints.api.Variable;
+import gov.nasa.jpf.constraints.expressions.functions.Function;
 import gov.nasa.jpf.constraints.util.ExpressionUtil;
-import java.util.ArrayList;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
+
+import java.util.*;
 
 public class SMTProblem {
   public List<Expression<Boolean>> assertions;
   public Set<Variable<?>> variables;
+  public Map<String, Function> functions;
```

Comment:
Function is used as a rawtype here

```suggestion
  public Map<String, Function<?>> functions;
```